project/profiles/capacity4more/behat/features/homepage.feature

```diff
@@ -64,8 +64,8 @@ Feature: Test homepage content and blocks
     Then  I should not see "Suggested Groups"
 
   @api
-  Scenario: Logged in, non member user should see "Suggested Groups" block
-    Given I am logged in as user "president"
```

Comment:
@anvmn 
if you're already here then add a test with _president_ that verify we do not see the block